### PR TITLE
Portfolio week 1

### PR DESCRIPTION
**Pull request by euskardi**
Sorry for the big Pull Request but I didn't see the advice to make little pull requests until the final slide

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,12 +8,36 @@
...
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Jesús Urquídez Calvo</h1>
+        <p>I'm a mexican of 19 years old that love Computer Science and Videogames...and a little bit weirdo</p>
```
Comment (renguyen) [inline portfolio/src/main/webapp/index.html:12]: Who isn't a little bit weird? :)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,12 +8,36 @@
...
+    </table>
+
+    <div>
+        <img width="100" src="/images/me_1.jpg">
+        <img width="100" src="/images/me_2.jpg">
+        <img width="100" height="100" src="/images/me_3.jpeg">    
+    </div>
+
+    <div>
+        <a href="nkedin.com/in/jesus-urquidez-calvo-517668194/"><img src="https://img.icons8.com/plasticine/100/000000/linkedin.png"/></a>
```
Comment (renguyen) [inline portfolio/src/main/webapp/index.html]:
```suggestion
        <a href="linkedin.com/in/jesus-urquidez-calvo-517668194/"><img src="https://img.icons8.com/plasticine/100/000000/linkedin.png"/></a>
```

Is this URL incomplete?

**Review by renguyen — COMMENTED** [review]
Great job finishing this! And no worries, this actually isn't too big. We can discuss separately how to appropriately size a PR :) I like how you were very careful with your styling by making sure that things are indented nicely!

Overall, looks great! One minor comment about a URL. My main suggestion is to make your PR title and description more descriptive of the changes that you're making. For the purpose of practice, feel free to assume that I'm not familiar with this portfolio project. For this PR, the title could be "Add personalized text and styling to portfolio page" and the description can be bullet pointed with a brief overview of your changes like:

- Update random quotes
- Add text about me
- Include photos

Also, screenshots are very helpful and show that you tested your code by bringing up a demo and making sure it looks as intended!

I'll approve after you look at my comment about the LinkedIn URL :)

**Comment by euskardi** [comment]
Sorry for the late I didn't see the comments, I already accepted the change of LinkedIn

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (euskardi) [reply portfolio/src/main/webapp/index.html:12]: everyone is weird in their own way xD

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (euskardi) [reply portfolio/src/main/webapp/index.html]: Yes, I didn't see the error

Comment (euskardi) [reply portfolio/src/main/webapp/index.html]: Even so, it causes an error, I just noticed that it is due to the lack of the HTTP protocol

**Review by renguyen — APPROVED** [review]

**Comment by renguyen** [comment]
Approved + merged! :)

**Pull request merged**